cves/kernel/CVE-2020-29370.yml

```diff
@@ -55,7 +55,14 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
+  This is a race condition vulnerability that causes a flaw memory 
+  corruption in the Linux Kernal Slub memory allocated subsystem. A race condition is
+  where two or more processes are using the same resources at the same time, so when 
+  TID -- a unique thread identifier -- wasn't incremented when the system is running, 
+  so two threads could be allocated to twice. This allows corruption of memory and 
```

Comment:
"two threads could be allocated to twice" ? 
Is this missing a word or 2?

Reply:
whoops yeah sorry ;w;